src/components/portfolio/DetailsMainDividends.tsx

```diff
@@ -97,7 +103,14 @@ const InfoBlock: React.FC<InfoBlockProps> = ({ title, children, infoText }) => {
   );
 };
 
-// returns the details page header
+/**
+ * This component represent the dividend section of the prtfolio details page.
+ * It consists of one larger historic chart, a dividend yield, a dividend payout ratio graph
```

Comment:
you could use {@link COMPONENT_NAME} to link to these sub components but that's just a suggestion